core/mondoo-linux-security.mql.yaml

```diff
@@ -1136,16 +1136,16 @@ queries:
         ```
     query: |
       if( file("/boot/grub2/grub.cfg" ).exists) {
-        file("/boot/grub2/grub.cfg").content.lines.where( _ == /^[^#]/ ).contains("audit\=(\s+)?1")
+        file("/boot/grub2/grub.cfg").content.contains('audit=1')
       }
       if( file("/boot/grub/grub.cfg").exists ) {
-        file("/boot/grub/grub.cfg").content.lines.where( _ == /^[^#]/ ).contains("audit\=(\s+)?1")
+        file("/boot/grub/grub.cfg").content.contains('audit=1')
       }
       if( file("/boot/grub/grub.conf").exists ) {
-        file("/boot/grub/grub.conf").content.lines.where( _ == /^[^#]/ ).contains("audit\=(\s+)?1")
+        file("/boot/grub/grub.conf").content.contains('audit=1')
```

Comment:
Same here: 
`file("/boot/grub2/grub.conf").content.lines.where( _ == /^[^#]/ ).any(_ == /audit(\s+)?\=(\s+)?1/)` 
@micheelengronne, how about this?